client/src/views/Home.vue

```diff
@@ -250,7 +253,9 @@ export default {
                     v-on="on"
                     text
                     small
-                    :disabled="selectedEligibleClips.length < 1"
+                    :disabled="
+                      selectedEligibleClips.length < 1 || !pipelines.length
+                    "
```

Comment:
if this line is so long it has to wrap, probably best to refactor as a computed property.